tests/server.py

```diff
@@ -739,9 +741,16 @@ def setup_test_homeserver(
             },
         }
     else:
+        if SQLITE_TEST_DB_LOCATION != ":memory:":
+            # nuke the DB on disk
+            try:
+                os.remove(SQLITE_TEST_DB_LOCATION)
+            except FileNotFoundError:
+                pass
```

Comment:
This feels a little icky, it seems for psql we generate a unique db name for each test. I'm not sure that's really doable here though.

Would it make more sense to not provide a location but just provide that, yes, you want to use an on-disk database and we put it in a known location inside of `_trial_temp`?

Reply:
> This feels a little icky,

agreed---icky, but hopefully useful? I'm mostly intending for this to be used when debugging one specific test interactively to try.

> we put it in a known location inside of _trial_temp

That could work---though I think if you have multiple instances of `trial` running then you end up with `_trial_temp`, `_trial_temp-1`, ... . (This occasionally comes up when I'm debugging one test in PyCharm while running another---the two modes are independent.)

Reply:
I meant "wherever trial thinks the root of my current test running is", not `_trial_temp` specifically, sorry for not being clear!

Reply:
roger that. I was worried at first that this would be hard to find, but it appears to just be the CWD when we run the tests.